domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -477,4 +488,54 @@ class StateRetriever @Inject constructor(
     }
     return StringList.newBuilder().addAllHtml(list).build()
   }
+
+  private fun parseImageWithRegions(jsonObject: JSONObject): ImageWithRegions {
```

Comment:
Do i need to add kdoc for all these methods?

Reply:
I don't think so--these methods are pretty straightforward and none are part of the public API of this class.